images/manageiq-base/Dockerfile

```diff
@@ -44,8 +44,8 @@ COPY container-assets/create_local_yum_repo.sh /
 RUN curl -L https://releases.ansible.com/ansible-runner/ansible-runner.el8.repo > /etc/yum.repos.d/ansible-runner.repo
 
 RUN dnf -y --disableplugin=subscription-manager install \
-      http://mirror.centos.org/centos/8/BaseOS/${ARCH}/os/Packages/centos-repos-8.1-1.1911.0.8.el8.${ARCH}.rpm \
-      http://mirror.centos.org/centos/8/BaseOS/${ARCH}/os/Packages/centos-gpg-keys-8.1-1.1911.0.8.el8.noarch.rpm \
+      http://mirror.centos.org/centos/8/BaseOS/x86_64/os/Packages/centos-repos-8.2-2.2004.0.1.el8.x86_64.rpm \
+      http://mirror.centos.org/centos/8/BaseOS/x86_64/os/Packages/centos-gpg-keys-8.2-2.2004.0.1.el8.noarch.rpm \
```

Comment:
I think we should change`http://mirror.centos.org/centos/8/` to`http://mirror.centos.org/centos/8.2.2004/`. This way, the rpms are guaranteed to exist when CentOS 8.3 is released, which will allow us to decide when to update to CentOS 8.3 and not because build fails.

Reply:
Also, is it intentional that it's no longer using `${ARCH}`?

Reply:
Nope, copy/paste error.